R/mobr_boxplots.R

```diff
@@ -64,11 +66,16 @@ calc_biodiv_single_group <- function(abund_vec, n_rare){
    if (length(abund_vec) > 0){
       out_vec["N"] <- sum(abund_vec)
       out_vec["S"] <- length(abund_vec)
-      if (!is.na(n_rare))
-         out_vec["S_rare"] <- rarefaction(abund_vec, method = "indiv", effort = n_rare)
-      out_vec["PIE"] <- calc_PIE(abund_vec)
-      out_vec["ENS_PIE"] <- 1/(1 - out_vec["PIE"])
       
+      inext1 <- iNEXT::iNEXT(abund_vec, q = 0, datatype = "abundance", size = n_rare, se = F)
+      out_vec["S_rare1"] <- inext1$iNextEst$qD[1] 
```

Comment:
Is there a good reason to use `iNEXT` to compute rarefied S instead of our function `rarefaction`?